kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/structure/SchemaCompilation.kt

```diff
@@ -461,4 +453,10 @@ class SchemaCompilation(
             transformation = transformation as Transformation<T, R>?
         )
     }
+
+    private fun typenameField(functionWrapper: FunctionWrapper<String>) = Field.Function(
```

Comment:
This definition skips all previous stuff done in `handleOperation` but IMHO that previous stuff is not necessary for the `__typename`: We already know the return type and we know that there are no input values.